api/v1alpha1/storageconsumer_types.go

```diff
@@ -72,6 +72,10 @@ type ClientStatus struct {
 	PlatformVersion string `json:"platformVersion"`
 	// StorageClient Operator Version
 	OperatorVersion string `json:"operatorVersion"`
+	// ClusterID is the id of the openshift cluster
+	ClusterID string `json:"clusterId"`
+	// StorageClientNamespacedName is the name and namespace of the StorageClient
+	StorageClientNamespacedName string `json:"storageClientNamespacedName"`
```

Comment:
This field is defined inside the consumer status inside the client's section so I think the `StorageClient` prefix is an overkill. I would suggest calling this field `NamespacedName` so accessing it would look like:
`consumer.Status.Client.NamespacedName`